grid-client/workloads/k8s.go

```diff
@@ -298,3 +305,18 @@ func (k *K8sNode) zosWorkload(cluster *K8sCluster, isWorker bool) (K8sWorkloads
 
 	return K8sWorkloads
 }
+
+func (k *K8sCluster) validateFlist() error {
```

Comment:
I think we validate the flists for master and workers, this function assigns the flist

Reply:
it also validate that either the cluster flist or the master flist exists

Reply:
I think we should validate the cluster flist only, others are validated,
Also we should display warning for example if cluster, master and workers flists don't match